src/utils/handleUserConfig.ts

```diff
@@ -72,9 +73,15 @@ export const handleInitData = (
             livePreviewObject.cleanCslpOnProduction ??
             config.cleanCslpOnProduction;
 
-        config.renderCslpButtonsByDefault =
-            livePreviewObject.renderCslpButtonByDefault ??
-            config.renderCslpButtonsByDefault;
+        config.editButton = {
+            enable: shouldRenderEditButton(
+                livePreviewObject.editButton ?? config.editButton
+            ),
+            exclude:
+                livePreviewObject.editButton?.exclude ??
+                config.editButton.exclude ??
```

Comment:
We could check if the exclude is an array.

Reply:
added extra array check